### ISSUE TITLE
Cannot register on the platform. No email received!

### ISSUE THREAD

**Issue by gcabezas** [CloudOpting/cloudopting-manager#269]

**Comment by xeviscc** [comment]
```console
[WARN] eu.cloudopting.service.MailService - E-mail could not be sent to user 'xavier.cases.camats@gmail.com', exception is: Authentication failed; nested exception is javax.mail.AuthenticationFailedException: 534-5.7.14 <https://accounts.google.com/signin/continue?sarp=1&scc=1&plt=AKgnsbsf
534-5.7.14 rkslngNbE_I-AuDDjeVfjQyUjvKtrjEVj2kl6S85zi-O4LaZiosvve3I0BmvhkqX6NOMB9
534-5.7.14 lZbu1BNf6Ungb-rktUXNgqvufMrblCBmzFOijsUerDpl83x3L0ThltIrMSrBQqKEtoY3UO
534-5.7.14 tpRI0gGNkRPDCj753516UzccNjJwNu16xmURx2EFmGgl7mj5Ygxn32UpaZcDribtaLBdBR
534-5.7.14 uhaGaDUByho3t6zBo_9y2DIKgve6U> Please log in via your web browser and
534-5.7.14 then try again.
534-5.7.14  Learn more at
534 5.7.14  https://support.google.com/mail/answer/78754 c128sm28832978wma.11 - gsmtp

[root@lab1 ~]# WARN] eu.cloudopting.service.MailService - E-mail could not be sent to user 'xavier.cases.camats@gmail.com', exception is: Authentication failed; nested exception is javax.mail.AuthenticationFailedException: 534-5.7.14 <https://accounts.google.com/signin/continue?sarp=1&scc=1&plt=AKgnsbsf
-bash: WARN]: no se encontró la orden
```

**Comment by xeviscc** [comment]
Time to time google needs that the account for cloudopting unlock captcha autentication at https://accounts.google.com/DisplayUnlockCaptcha

Other problems may occur. Check here: https://support.google.com/mail/answer/14257?rd=1

If we want to avoid this kind of problems we may think about geting a "real" email.

**Comment by guidospadotto-profesia** [comment]
I am facing this issue, which prevents me from testing my changes. 
Reopening as it has not been solved yet.

**Comment by xeviscc** [comment]
Waitiing for a proper mail account to proceed.

**Comment by xeviscc** [comment]
We have the information of the new e-mail in the e-mail list. Please integrate.

**Comment by guidospadotto-profesia** [comment]
Done in 6dc4a91

**Comment by xeviscc** [comment]
@gcabezas Could you please test it and close this issue if the problem is solved?

**Comment by xeviscc** [comment]
Error sending mails.

```console
[DEBUG] com.sun.mail.smtp - exception reading response
java.net.SocketException: Connection reset
        at java.net.SocketInputStream.read(SocketInputStream.java:209) ~[na:1.8.0_66-internal]
        at java.net.SocketInputStream.read(SocketInputStream.java:141) ~[na:1.8.0_66-internal]
        at com.sun.mail.util.TraceInputStream.read(TraceInputStream.java:124) ~[javax.mail-1.5.5.jar:1.5.5]
        at java.io.BufferedInputStream.fill(BufferedInputStream.java:246) ~[na:1.8.0_66-internal]
        at java.io.BufferedInputStream.read(BufferedInputStream.java:265) ~[na:1.8.0_66-internal]
        at com.sun.mail.util.LineInputStream.readLine(LineInputStream.java:92) ~[javax.mail-1.5.5.jar:1.5.5]
        at com.sun.mail.smtp.SMTPTransport.readServerResponse(SMTPTransport.java:2331) [javax.mail-1.5.5.jar:1.5.5]
        at com.sun.mail.smtp.SMTPTransport.openServer(SMTPTransport.java:2078) [javax.mail-1.5.5.jar:1.5.5]
        at com.sun.mail.smtp.SMTPTransport.protocolConnect(SMTPTransport.java:699) [javax.mail-1.5.5.jar:1.5.5]
        at javax.mail.Service.connect(Service.java:366) [javax.mail-1.5.5.jar:1.5.5]
        at org.springframework.mail.javamail.JavaMailSenderImpl.connectTransport(JavaMailSenderImpl.java:501) [spring-context-support-4.2.4.RELEASE.jar:4.2.4.RELEASE]
        at org.springframework.mail.javamail.JavaMailSenderImpl.doSend(JavaMailSenderImpl.java:421) [spring-context-support-4.2.4.RELEASE.jar:4.2.4.RELEASE]
        at org.springframework.mail.javamail.JavaMailSenderImpl.send(JavaMailSenderImpl.java:345) [spring-context-support-4.2.4.RELEASE.jar:4.2.4.RELEASE]
        at org.springframework.mail.javamail.JavaMailSenderImpl.send(JavaMailSenderImpl.java:340) [spring-context-support-4.2.4.RELEASE.jar:4.2.4.RELEASE]
        at eu.cloudopting.service.MailService.sendEmail(MailService.java:104) [common-component-1.0-SNAPSHOT.jar:1.0-SNAPSHOT]
        at eu.cloudopting.service.MailService.sendActivationEmail(MailService.java:120) [common-component-1.0-SNAPSHOT.jar:1.0-SNAPSHOT]
        at eu.cloudopting.service.MailService$$FastClassBySpringCGLIB$$ef84f7bd.invoke(<generated>) [common-component-1.0-SNAPSHOT.jar:1.0-SNAPSHOT]
        at org.springframework.cglib.proxy.MethodProxy.invoke(MethodProxy.java:204) [spring-core-4.2.4.RELEASE.jar:4.2.4.RELEASE]
        at org.springframework.aop.framework.CglibAopProxy$CglibMethodInvocation.invokeJoinpoint(CglibAopProxy.java:720) [spring-aop-4.2.4.RELEASE.jar:4.2.4.RELEASE]
        at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:157) [spring-aop-4.2.4.RELEASE.jar:4.2.4.RELEASE]
        at org.springframework.aop.interceptor.AsyncExecutionInterceptor$1.call(AsyncExecutionInterceptor.java:108) [spring-aop-4.2.4.RELEASE.jar:4.2.4.RELEASE]
        at java.util.concurrent.FutureTask.run(FutureTask.java:266) [na:1.8.0_66-internal]
        at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142) [na:1.8.0_66-internal]
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617) [na:1.8.0_66-internal]
        at java.lang.Thread.run(Thread.java:745) [na:1.8.0_66-internal]
[WARN] eu.cloudopting.service.MailService - E-mail could not be sent to user 'gcabezasr@bcn.cat', exception is: Mail server connection failed; nested exception is javax.mail.MessagingException: Exception reading response;
  nested exception is:
        java.net.SocketException: Connection reset. Failed messages: javax.mail.MessagingException: Exception reading response;
  nested exception is:
        java.net.SocketException: Connection reset
```

**Comment by guidospadotto-profesia** [comment]
I have set the paramers as provided by @ignasijr but this is what I get:
> [INFO] javax.mail - JavaMail version 1.5.5
> [INFO] javax.mail - successfully loaded resource: /META-INF/javamail.default.providers
> [INFO] javax.mail - Tables of loaded providers
> [INFO] javax.mail - Providers Listed By Class Name: {com.sun.mail.smtp.SMTPSSLTransport=javax.mail.Provider[TRANSPORT,smtps,com.sun.mail.smtp.SMTPSSLTransport,Oracle], com.sun.mail.smtp.SMTPTransport=javax.mail.Provider[TRANSPORT,smtp,com.sun.mail.smtp.SMTPTransport,Oracle], com.sun.mail.imap.IMAPSSLStore=javax.mail.Provider[STORE,imaps,com.sun.mail.imap.IMAPSSLStore,Oracle], com.sun.mail.pop3.POP3SSLStore=javax.mail.Provider[STORE,pop3s,com.sun.mail.pop3.POP3SSLStore,Oracle], com.sun.mail.imap.IMAPStore=javax.mail.Provider[STORE,imap,com.sun.mail.imap.IMAPStore,Oracle], com.sun.mail.pop3.POP3Store=javax.mail.Provider[STORE,pop3,com.sun.mail.pop3.POP3Store,Oracle]}
> [INFO] javax.mail - Providers Listed By Protocol: {imaps=javax.mail.Provider[STORE,imaps,com.sun.mail.imap.IMAPSSLStore,Oracle], imap=javax.mail.Provider[STORE,imap,com.sun.mail.imap.IMAPStore,Oracle], smtps=javax.mail.Provider[TRANSPORT,smtps,com.sun.mail.smtp.SMTPSSLTransport,Oracle], pop3=javax.mail.Provider[STORE,pop3,com.sun.mail.pop3.POP3Store,Oracle], pop3s=javax.mail.Provider[STORE,pop3s,com.sun.mail.pop3.POP3SSLStore,Oracle], smtp=javax.mail.Provider[TRANSPORT,smtp,com.sun.mail.smtp.SMTPTransport,Oracle]}
> [INFO] javax.mail - successfully loaded resource: /META-INF/javamail.default.address.map
> [DEBUG] javax.mail - getProvider() returning javax.mail.Provider[TRANSPORT,smtp,com.sun.mail.smtp.SMTPTransport,Oracle]
> [DEBUG] com.sun.mail.smtp - useEhlo true, useAuth true
> [DEBUG] com.sun.mail.smtp - trying to connect to host "vl23232.dns-privadas.es", port 465, isSSL false
> [DEBUG] com.sun.mail.smtp - exception reading response
> java.net.SocketException: Connection reset
> 	at java.net.SocketInputStream.read(SocketInputStream.java:209) ~[na:1.8.0_40]
> 	at java.net.SocketInputStream.read(SocketInputStream.java:141) ~[na:1.8.0_40]
> 	at com.sun.mail.util.TraceInputStream.read(TraceInputStream.java:124) ~[javax.mail-1.5.5.jar:1.5.5]
> 	at java.io.BufferedInputStream.fill(BufferedInputStream.java:246) ~[na:1.8.0_40]
> 	at java.io.BufferedInputStream.read(BufferedInputStream.java:265) ~[na:1.8.0_40]
> 	at com.sun.mail.util.LineInputStream.readLine(LineInputStream.java:92) ~[javax.mail-1.5.5.jar:1.5.5]
> 	at com.sun.mail.smtp.SMTPTransport.readServerResponse(SMTPTransport.java:2331) [javax.mail-1.5.5.jar:1.5.5]
> 	at com.sun.mail.smtp.SMTPTransport.openServer(SMTPTransport.java:2078) [javax.mail-1.5.5.jar:1.5.5]
> 	at com.sun.mail.smtp.SMTPTransport.protocolConnect(SMTPTransport.java:699) [javax.mail-1.5.5.jar:1.5.5]
> 	at javax.mail.Service.connect(Service.java:366) [javax.mail-1.5.5.jar:1.5.5]

Using the Non-SSL settings, it seems the mail service is able to connect but not to send the mail.

**Comment by guidospadotto-profesia** [comment]
I have also tried to talk with the SMTP server via telnet as reported here 
http://gerardnico.com/wiki/smtp_telnet
. No answer at all after "ehlo".
@ignasijr are you sure that you are able to send emails using the mail server you provided?

**Comment by guidospadotto-profesia** [comment]
Guidos-MacBook-Pro-2:cloudopting-manager guido$ telnet mail.cloudopting.eu 25
Trying 212.48.92.191...
Connected to cloudopting.eu.
Escape character is '^]'.
220-vl23232.dns-privadas.es ESMTP Exim 4.85 #2 Wed, 30 Mar 2016 16:52:42 +0200 
220-We do not authorize the use of this system to transport unsolicited, 
220 and/or bulk e-mail.
ehlo dev@cloudopting.eu
250-vl23232.dns-privadas.es Hello dev@cloudopting.eu [84.240.129.9]
250-SIZE 52428800
250-8BITMIME
250-PIPELINING
250-AUTH PLAIN LOGIN
250-STARTTLS
250 HELP
AUTH PLAIN <base64 of password>
435 Unable to authenticate at present

**Comment by ignasijr** [comment]
Yes I do, are you connecting with no SSL right?

**Comment by guidospadotto-profesia** [comment]
Fixed in 02bdb4f .
